pkg/server/server.go

```diff
@@ -417,15 +377,8 @@ func (s *Server) HTTPHandler() http.Handler {
 
 			// TODO remove multicluster
 			authHandlerWithUser(func(user *auth.User, w http.ResponseWriter, r *http.Request) {
-				cluster := serverutils.GetCluster(r)
```

Comment:
for the reviewer: I was not sure which chunk exactly is supposed to go away in MC scenarios. LMK if the whole thing can go away.